packages/@coorpacademy-components/src/molecule/dashboard/cards-list/index.js

```diff
@@ -309,31 +315,31 @@ class CardsList extends React.PureComponent {
       map(([key, card]) => {
         return (
           <div className={style.card} key={key}>
-            <Card {...card} dataName={`${dataName}-${key}`} />
+            {card && card.type === 'learningPriority' ? (
```

Comment:
Hi Adam, since CardsList have `type` props now, maybe we can use that in this condition instead and remove the `type` props for `LearningPriorityCard`?

Reply:
ah yeah, I pushed the update!

Reply:
Thank you! just one last minor thing, maybe we can remove the `type` from these

Reply:
https://github.com/CoorpAcademy/components/pull/2875/files#diff-6f3b76da892827e600e47cad725fdb68760d54c45588791f5fbef1dba7c8853aR6

Reply:
ah yes oups! I removed here: https://github.com/CoorpAcademy/components/pull/2875/commits/c517eae53f43250b01235babc04dc0ffaaa31988#diff-ae28aedce2e7aea5d988092ce6cd2d60f68633db53061240264b1f4c1c65963dR26 and from the types but I forgot in the others fixtures 👍

Reply:
it's done now